carbon14/django.py

```diff
@@ -86,3 +86,24 @@ def serialize(self, instance, *args, **kwargs):
         if isinstance(value, Point):
             value = tuple(value)
         return value
+
+
+class PaginatedModelCollection(ModelCollection):
+    def _resolve(
```

Comment:
I think that we can put this directly into ModelCollection.

Offset and limit can be none by default. 
And only if they are not none, they have an effect.